zenith/README.md

```diff
@@ -31,7 +31,7 @@ As of August 24 2023, the latest source code for Project Zenith is available as
 
 To get started:
 
-1. Visit the [draft pull request](https://github.com/dagger/dagger/pull/5443)
+1. Visit the [current draft pull request](https://github.com/dagger/dagger/pull/5757) and the [original draft PR (now closed)](https://github.com/dagger/dagger/pull/5443)
 2. Checkout the corresponding branch
```

Comment:
We actually merged that PR too as of a few hours ago 😄 So I actually think we can just remove `1.` and `2.` from here now.

Reply:
Moving fast!! I'll close this PR then, as it seems like the readme has moved on.